src/main/java/tc/oc/pgm/modules/MaxBuildHeightMatchModule.java

```diff
@@ -33,4 +34,23 @@ public void checkBuildHeight(BlockTransformEvent event) {
       }
     }
   }
+
+  private int getExtraHeight(Material material) {
```

Comment:
`getMaterialHeight` maybe? Should return 1 or 2.

Reply:
This is old code, review the latest commit only